gammapy/spectrum/tests/test_fit.py

```diff
@@ -152,47 +123,6 @@ def setup(self):
         )
 
 
-    def test_basic_results(self):
```

Comment:
All those tests are replaced by tests in test_dataset.py, right?

Reply:
Yes, I removed the duplication of tests between `test_fit.py` and `test_dataset.py`. The distinction is a bit arbitrary, but for now it shouldn't matter. I guess it's OK to clean up the tests in future.